articles/openshift/oshift-how-restrict-access-to-openshift-routes-by-ip.md

```diff
@@ -1,57 +1,65 @@
 ---
-title: UKCloud Knowledge Centre article about whitelisting public IP's on your openshift routes | UKCloud Ltd
-description: Restrict access to openshift routes by public IP[s]
+title: How to restrict access to a route in OpenShift by the source public IP address
+description: Restrict access to OpenShift routes by the source public IP address
 services: openshift
 author: Mudasar Hussain
 toc_rootlink: How To
 toc_sub1: 
 toc_sub2:
 toc_sub3:
 toc_sub4:
-toc_title: restrict access to openshift routes by public IP[s]
+toc_title: Restrict access to OpenShift routes by the source public IP address
 toc_fullpath: How To/restricting-access-to-openshift-routes-by-ip.md
 toc_mdlink: oshift-how-restrict-access-to-openshift-routes-by-ip.md
 ---
 
-# How to restrict access to a route in openshift by IP[s] address
+# How to only allow trusted IP's to be able to access routes in Openshift
 
 ## Overview
 
-This knowledge centre article explains how openshift customers can restrict access to the routes they create to their deployed services by IP[s] addresses.
+This knowledge centre article explains how OpenShift developers can restrict access to the routes they create to their deployed services in OpenShift by IP address. This can be done to increase the security of a route by only allowing trusted IP's to access the OpenShift route.
 
 ### Intended audience
 
-Openshift developers who are looking to secure exposed routes of application by IP[s]
+OpenShift developers who have already created and deployed services into OpenShift and also created routes for those services and exposed those routes. This article helps those users to further secure access to those exposed routes by restricting access to those exposed routes by only allowing trusted IP's to be able to access those routes.
```

Comment:
OpenShift developers who have created and deployed services into OpenShift, and created and exposed routes for those services. This article describes how to further secure access to those exposed routes by allowing only trusted IPs to access them.